ecs-init/volumes/types/types_test.go

```diff
@@ -44,6 +45,13 @@ func TestRemoveMount(t *testing.T) {
 		assert.False(t, v.RemoveMount("id"))
 		assert.Empty(t, v.Mounts)
 	})
+	t.Run("mount reference count is decremented", func(t *testing.T) {
+		v := &Volume{}
+		v.AddMount("id")
+		v.AddMount("id")
+		assert.True(t, v.RemoveMount("id"))
+		assert.Equal(t, map[string]int{"id": 1}, v.Mounts)
+	})
 	t.Run("mount should be removed if it exists", func(t *testing.T) {
```

Comment:
nit: For clarity, could we update the name of this test run to "mount should be removed if it exists and mount reference count is 1" under the new behavior? (i.e., mount should no longer be removed for solely just existing)

Reply:
Yep. Updated. Thanks!